### ISSUE TITLE
New vr::VRServerDriverHost()->SetRecommendedRenderTargetSize() API is broken

### ISSUE THREAD

**Issue by TheDeveloperGuy** [ValveSoftware/openvr#1445]
It's a great idea to allow the driver to change render target size, however this API doesn't actually do anything (ie. calling it with a new render target size does not cause SteamVR to use a new render target size). The change (if any) is not visible in the headset, and the render target size in SteamVR settings doesn't change.

**Comment by TheDeveloperGuy** [comment]
So, it seems some titles respond accordingly to render target size change during runtime, so it's not completely broken. However SteamVR "landing scene" and "Home" do not, neither does SteamVR settings.

**Comment by risa2000** [comment]
@JoeLudwig
In v1.15.12, when calling this API with parameters `newWidth` and `newHeight` the client API returns the new recommended resolution:
client `width` = `newHeight`
client `height` = `oldHeight`

For example:
If the driver initializes the recommended resolution to [width = 3936, height=2612] and scales it down a bit by calling `IVRServerDriverHost::SetRecommendedRenderTargetSize(3280, 2177)`, the new resolution returned by the client API (`IVRSystem::GetRecommendedRenderTargetSize`) is: width=2176, height=2612.

To sum up:
The `height` does not change and the `width` is set with the new `height`.
